extension/oauth2clientauthextension/extension_test.go

```diff
@@ -272,3 +276,60 @@ func TestOAuthExtensionShutdown(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Nil(t, oAuthExtensionAuth.Shutdown(context.Background()))
 }
+
+func TestFailContactingOAuth(t *testing.T) {
```

Comment:
NIT - if possible Please combine gRPC case and HTTP tests into single function by using `t.RUN()`